cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/data/DataLayer.java

```diff
@@ -46,7 +46,7 @@
 import org.apache.cassandra.spark.sparksql.filters.PartitionKeyFilter;
 import org.apache.cassandra.spark.sparksql.filters.PruneColumnFilter;
 import org.apache.cassandra.spark.sparksql.filters.SparkRangeFilter;
-import org.apache.cassandra.spark.stats.Stats;
+import org.apache.cassandra.analytics.stats.Stats;
```

Comment:
please rename package to `bridge` instead.

Reply:
I don't think that is really in scope for this change. I don't want PRs to become a bucket for every change everyone want's to squeeze in. I'm tempted to revert https://github.com/apache/cassandra-analytics/pull/88/commits/67590a268f095ef6f91bd7e1d195300cd75dac71 as well.

Reply:
There are plenty of util and standalone classes in `org.apache.cassandra.spark.reader` that could be used downstream, I'm not saying we can't relocate especially for major version change but it seems worth a standalone change.

Reply:
Reverting and relocating in a separate patch sounds good to me.